velox/exec/RowNumber.cpp

```diff
@@ -499,4 +471,60 @@ void RowNumber::spillInput(
   }
 }
 
+void RowNumber::addInputInternal(const RowVectorPtr& input, bool fromSpill) {
+  // TODO: Add ensureInputFits.
```

Comment:
Why is this a TODO?

Reply:
Got it, I will add both `ensureInputFits` and `yield` in this PR.